scene/gui/code_edit.h

```diff
@@ -457,6 +460,9 @@ class CodeEdit : public TextEdit {
 	Point2 get_delimiter_start_position(int p_line, int p_column) const;
 	Point2 get_delimiter_end_position(int p_line, int p_column) const;
 
+	Vector<String> get_block_key_delimiters() const;
+	void set_block_key_delimiters(const List<String> *p_delimiters);
```

Comment:
```suggestion
	void set_block_key_delimiters(const List<String> &p_delimiters);
```
No reason to pass it as a pointer unless it's optional, in that case it needs a null check 

But this should really use `Vector<String>` IMO, it transfers to a `Vector` anyway so there's going to be a conversion anyway, so I'd say:
```suggestion
	void set_block_key_delimiters(const Vector<String> &p_delimiters);
```

Reply:
I'm wondering if it is viable to convert it to a Vector, as the API requires a List from the key delimiters. So I need to convert it to a Vector and then pass it to the method. It seems to complicate it a little bit. What do you think?